packages/twenty-website/src/app/components/oss-friends/Header.tsx

```diff
@@ -0,0 +1,41 @@
+'use client';
```

Comment:
Any idea why there's not coloring by Github on that file? There's probably something wrong but I can't figure out what?

Reply:
The rest looks good to me. Haven't tried it locally yet. Does it behave well on mobile?

Reply:
Let me get you some screenshots @FelixMalfait

Reply:
Some mobile screenshots

![localhost_3000_oss-friends(iPhone 12 Pro)](https://github.com/twentyhq/twenty/assets/140154534/acdc7b8e-0a9b-4d31-96e1-4851c0112582)

![localhost_3000_oss-friends(Pixel 7) (2)](https://github.com/twentyhq/twenty/assets/140154534/31657e9f-2d31-493b-afbb-4ad0e2e90db6)

![localhost_3000_oss-friends(Pixel 7)](https://github.com/twentyhq/twenty/assets/140154534/a8ce218c-a89a-40e8-acde-5bc71a0818e7)

Reply:
@FelixMalfait I've not figured out why the syntax highlighting is not working for this particular file. The  file has no visible errors